roles/wildfly_validation/tasks/yaml_setup.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: "Created a required directory"
+  ansible.builtin.command: >
+    mkdir -p {{ wildfly_home }}/modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/
+
+- name: "enable that feature you need to add a ServiceLoader configuration"
+  ansible.builtin.command: >
+    echo 'org.jboss.as.controller.persistence.yaml.YamlConfigurationExtension' > {{ wildfly_home }}/modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/org.jboss.as.controller.persistence.ConfigurationExtension
```

Comment:
@RanabirChakraborty I think the two first tasks (create directory, enable feat) are left over from a copy'n'paste.